src/github.rs

```diff
@@ -82,6 +82,9 @@ fn headers() -> Result<HeaderMap, InvalidHeaderValue> {
     Ok(headers)
 }
 
+/// Lookup information about the given commit reference from GitHub.
+/// If it is a commit forked from the "mainline" git history (e.g. a beta/stable tag),
```

Comment:
When I tried, I couldn't find a way to specify beta. --start=1.89.0 didn't work. Nor did --start=beta. So are you sure about the "beta tag" bit?

Reply:
Hmm, good point, beta releases don't actually have tags in git. Reworded.